ISBN biznis (#504)
exercism/julia: exercises/practice/isbn-verifier/.docs/instructions.md

## Bonus tasks

> If you attempt these tasks, write your own tests for them and include those in your solution too!

Reviewer: I'm not sure this will work with the web editor. Perhaps we can specify they should be in a separate file or in the main solution?

Author: Works in the web editor for me. Try adding `@test false` at the end of an exercise and then running the tests in the web editor.

Author: We include the student's solution file within a custom testset so it will all work. It'll even work if the student adds `using Test` because they think they need it or a linter tells them to.

Reviewer: > Works in the web editor for me. Try adding @test false at the end of an exercise and then running the tests in the web editor.

Ah yea, that works. I was thinking about adding it to the test file itself.

Author: ```suggestion
If you attempt these tasks please write your own tests for them in your solution file!
```

Clearer?

Reviewer: Yes, definitely!

Author: Thanks :)
